estate_account/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'Estate Account',
+    'depends': ['estate', 'account'],
+    'summary': 'Enables invoicing for Estate properties',
```

Comment:
```suggestion
    'summary': "Enables invoicing for Estate properties",
```
Key should be in single quotes whereas values in double quotes.
In case of technical terms it should be in `''`.